src/UI/Seller/src/app/products/components/product-edit/product-edit.component.ts

```diff
@@ -100,7 +100,7 @@ export class ProductEditComponent implements OnInit, OnDestroy {
   isCreatingNew: boolean
   @Input()
   dataIsSaving = false
-  userContext: any = {}
+  userContext: UserContext = {} as any
```

Comment:
Do you still get intellisense when you try to access properties off of userContext when you cast to type `any`? If so, no change is needed otherwise, is it possible to cast to type `UserContext`?